account/src/main/java/com/whz/account/resource/AccountResource.java

```diff
@@ -53,42 +83,69 @@ public AccountResource(final Stage stage) {
+   *     successfully created
+   */
   public Completes<Response> handlePostAccount(AccountData data) {
     return Account.defineWith(stage, data.balance)
         .andThenTo(
-            state ->
-                Completes.withSuccess(
-                    Response.of(
-                        Created,
-                        headers(of(Location, location(state.id)))
-                            .and(of(ContentType, "application/json")),
-                        serialized(AccountData.from(state)))));
+            state -> {
+              createPortfolioRequest(state.id);
+              return Completes.withSuccess(
+                  Response.of(
+                      Created,
+                      headers(of(Location, location(state.id)))
+                          .and(of(ContentType, "application/json")),
+                      serialized(AccountData.from(state))));
+            });
   }
 
-  /*
-   * // PUT public Completes<Response> handlePut(AccountData data) { return
-   * Completes.withSuccess(Response.of(Ok, "Updated")); }
-   *
-   * // DELETE public Completes<Response> handleDelete(String id) { return
-   * Completes.withSuccess(Response.of(Ok, "Deleted")); }
-   */
-
+  /** Maps the HTTP-Requests to the corresponding handler method. */
   @Override
   public Resource<?> routes() {
     return resource(
         getClass().getSimpleName(),
         get("/account/{id}").param(String.class).handle(this::handleGetAccount),
         post("/account").body(AccountData.class).handle(this::handlePostAccount));
-    // put("/account").body(AccountData.class).handle(this::handlePut),
-    // delete("/account/{id}").param(String.class).handle(this::handleDelete));
   }
 
+  /**
+   * Parses a String to represent our account id path.
+   *
+   * @param id - the id of the AccountState object
+   * @return String - the parsed id
+   */
   private String location(final String id) {
     return "/account/" + id;
   }
 
-  private Completes<Account> resolve(final String id) {
-    return stage.actorOf(Account.class, stage.addressFactory().from(id), AccountEntity.class);
+  /**
+   * The full HTTP POST-Request call to the Portfolio microservice which will create a Portfolio to
+   * our Account in a 1:1 relation based on the given id.
+   *
+   * @param id - the id which will be shared between an Account and a Portfolio
+   */
+  private void createPortfolioRequest(String id) {
+    String requestBody = id;
+
+    HttpClient client = HttpClient.newHttpClient();
+    HttpRequest request =
+        HttpRequest.newBuilder()
+            .uri(URI.create(portfolioUrl + "/portfolio"))
+            .POST(HttpRequest.BodyPublishers.ofString(requestBody))
+            .build();
+
+    try {
+      client.send(request, HttpResponse.BodyHandlers.ofString());
+    } catch (IOException | InterruptedException e) {
+      logger.error("Unable to send HTTP Request", e);
+    }
```

Comment:
Just a note that does not need to be implemented: For the small use case, this implementation is sufficient, but if the project should be extended, it would make sense to write a separate API class for this.